index.bs

```diff
@@ -6064,6 +6080,207 @@ However, [=authenticators=] that do not utilize [[!FIDO-CTAP]] do not necessaril
+
+If the [=client device=] is incapable of generating a [=hardware-bound device key pair=], or the registration or authentication operation fails for any reason, this extension is ignored and no [=hardware-bound device key pair=] is created. In this case, there is no [=devicePubKey=] extension output generated.
+
+The [=hardware-bound device key pair=] is not on its own a [=user credential=] and does not have its own [=credential ID=]. Instead, the returned [=device public key=] is a device-specific contextual attribute of its associated [=user credential=]. That is, when that [=user credential=] is used&mdash;along with the [=devicePubKey=] extension&mdash;on a particular [=client device=], a particular [=device public key=] is returned by the extension, along with a signature demonstrating proof-of-possession of the [=device private key=] by that device.
+
+
+### Relying Party Usage ### {#sctn-device-publickey-extension-usage}
+
+Issue(w3c/webauthn#1665): TODO: explicitly add the notion of "Synced/multi-device User Credentials" to the spec. This will involve adding proper definitions for "single-device (user) credentials" and "multi-device (user) credentials" in the terminology section, updating the text of this "Device-bound public key extension" section, and perhaps other approapriate changes or additions to other portions of the spec.
+
+This extension is intended for use by those [=[RPS]=] employing risk-analysis systems informing their sign-in decisions. This extension provides a "device continuity" signal <i>when used consistently</i> with both {{CredentialsContainer/create()|navigator.credentials.create()}} and {{CredentialsContainer/get()|navigator.credentials.get()}} operations:
+
+A new signature by a new [=device-bound key=] ("sig 1 + dpk 1") is returned to the [=[RP]=], along with a new [=user credential=], when the `devicePubKey` extension is used with a {{CredentialsContainer/create()|navigator.credentials.create()}} call on a given device ("device 1"). For example, as part of a [=registration ceremony=]. Further signatures by the <i>same</i> [=device-bound key=] ("sig n + dpk 1") are returned to the [=[RP]=] along with [=assertions=] generated using the latter user credential when the [=[RP]=] subsequently invokes {{CredentialsContainer/get()|navigator.credentials.get()}} calls including the `devicePubKey` extension, on that <i>same device</i> ("device 1") upon which the user credential was created. This behavior (on "device 1") is independent of whether this user credential has been copied to any other devices.
```

Comment:
```
Further signatures by the <i>same</i> [=device-bound key=] ("sig n + dpk 1") are returned to the [=[RP]=] along with [=assertions=] generated using the latter user credential
```

This "latter" threw me off as there aren't any other user credentials being discussed. Would it suffice to say just "generated using the user credential", or maybe "generated using the same user credential"?

Reply:
Ok, let's give "generated using the same user credential" a try, thx.  I also did some other wordsmithing while I was in there, see 619ebb9.

Reply:
That's certainly better! But now when I read these paragraphs again I found them a bit hard to follow, I think mostly because they're "temporally reversed" (effects are described before their causes). I took a stab at shortening and rearranging it a bit, what do you think?

(also added some non-breakable spaces, because line breaks there look really bad :slightly_smiling_face:)

```
When the `devicePubKey` extension is used with a {{CredentialsContainer/create()|create()}} call
on a given [=client device=] ("device&nbsp;1") to create a new [=user credential=],
a new signature ("sig&nbsp;1") by a new [=device-bound key=] ("dpk&nbsp;1") is returned along with the new [=device public key=].
In subsequent {{CredentialsContainer/get()|get()}} operations
using the same user credential on that <i>same [=client device=]</i> ("device&nbsp;1"),
the [=assertions=] include further signatures by the <i>same</i> [=device-bound key=] ("sig&nbsp;n + dpk&nbsp;1").
This behavior on "device&nbsp;1" is independent of whether this user credential has been copied to any other [=client devices=].

Then, if this same user credential is copied or moved to a different [=client device=] ("device&nbsp;2"),
the initial {{CredentialsContainer/get()|get()}} call on "device&nbsp;2" (that includes the `devicePubKey` extension)
will produce an [=assertion=] that includes <strong>a new signature by a <i>new</i> [=device-bound key=]</strong> ("sig&nbsp;x + dpk&nbsp;2").
Note that such a "multi-device user credential" can be exercised on "device&nbsp;2"
without a {{CredentialsContainer/create()|create()}} having been performed on "device&nbsp;2".
Subsequent {{CredentialsContainer/get()|get()}} calls on "device&nbsp;2",
using the `devicePubKey` extension and the same user credential, yield further signatures by "dpk&nbsp;2".
```

Reply:
Thanks, that's a good suggestion. I adopted and massaged it some in 2730294 :)